packages/alfa-dom/src/h.ts

```diff
@@ -104,6 +123,19 @@ export namespace h {
     );
   }
 
+  export function shadow(
+    children: Array<Node | string>,
+    mode: Shadow.Mode = Shadow.Mode.Open,
+    style: Array<Sheet> = []
```

Comment:
```suggestion
    mode: Shadow.Mode,
    children: Array<Node | string>,
    style: Array<Sheet> = []
```

Matches `Shadow.of()` argument order.

Reply:
🤔 That was a conscious choice because I guess that our main use case won't care about the mode and I wanted to make it optional.
I've overloaded the signature instead, so we can have the quick `h.shadow([foo, bar])` which I guess we'll use often.